app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_cat_and_dog_ages_are_zeros():
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_cat_and_dog_ages_give_zero_human_age():
+    assert get_human_age(10, 12) == [0, 0]
+
+
+def test_cat_and_dog_ages_give_one_human_age():
```

Comment:
Why these two tests have same implementation?

Reply:
ctrl+c ctrl+v went wrong) 
thanks that notice